semcore/date-picker/src/components/RangePickerAbstract.jsx

```diff
@@ -102,25 +103,34 @@ class RangePickerAbstract extends Component {
   handlerKeyDown = (e) => {
     if (e.target !== e.currentTarget) return;
     const { displayedPeriod, highlighted, preselectedValue } = this.asProps;
-    const day = this.keyDiff[e.keyCode];
+    const day = this.keyDiff[e.code];
 
     const setNextDisplayedPeriod = (next_highlighted) => {
-      const [, right_period] = next_highlighted;
-
-      if (right_period) {
-        const month_right_period = right_period.getMonth();
-        const month_displayed_Period = displayedPeriod.getMonth();
-        if (month_right_period - month_displayed_Period > 1) {
-          return RangePickerAbstract.subtract(right_period, 1, 'month');
-        } else if (month_right_period - month_displayed_Period < 0) {
-          return right_period;
+      const [left_period, right_period] = next_highlighted;
+
+      const monthDisplayedPeriod = displayedPeriod?.getMonth();
+      const monthPeriod = right_period || left_period;
```

Comment:
I understand that `left_period` corresponds to date of the highlighted period start, while `right_period` for the end. What `monthPeriod` means in this context? And why are making such a fallback?

Reply:
renamed to `period`. 

Fallback - for move calendar if we try to select first point of period.